frontend/src/components/auth/Login.tsx

```diff
@@ -24,55 +48,66 @@ const Login = (): React.ReactElement => {
     setAuthenticatedUser(user);
   };
 
-  const onSignUpClick = () => {
-    history.push(SIGNUP_PAGE);
-  };
-
   if (authenticatedUser) {
     return <Redirect to={HOME_PAGE} />;
   }
 
-  return (
-    <div style={{ textAlign: "center" }}>
-      <h1>Login</h1>
-      <form>
-        <div>
-          <input
-            type="email"
-            value={email}
-            onChange={(event) => setEmail(event.target.value)}
-            placeholder="username@domain.com"
-          />
-        </div>
-        <div>
-          <input
-            type="password"
-            value={password}
-            onChange={(event) => setPassword(event.target.value)}
-            placeholder="password"
-          />
-        </div>
-        <div>
-          <button
-            className="btn btn-primary"
-            type="button"
-            onClick={onLogInClick}
+  const title = isAdmin ? "Admin Login" : "Teacher Login";
+  const image = isAdmin ? ADMIN_SIGNUP_IMAGE : TEACHER_SIGNUP_IMAGE;
+  const form = (
+    <VStack width="50%" gap={4}>
```

Comment:
```suggestion
    <VStack width="75%" gap={4}>
```
nit: match figma a little closer